app/models/issue.rb

```diff
@@ -44,16 +45,25 @@ class Issue < ApplicationRecord
   before_validation :set_reviewed, on: :update
   before_save :set_expected_closure, if: :status_changed?
 
-  def to_s
-    "#{Issue.human_enum_name(:kind, kind)} ##{id}"
+  scope :not_archived, -> { where(archived_at: nil) }
+  scope :open, -> { where(status: %w[received reviewed]) }
+
+  def self.not_approved
+    includes(:photos).where(description_status: %i[internal deleted], photo: { status: %i[internal deleted] })
   end
 
-  def map_icon
-    "icons/map/active/png/#{category&.kind || 'blank'}-#{icon_color}.png"
+  def self.unsupported
+    eager_load(:supporters).where(kind: 'idea').where(supporters_sql)
   end
 
-  def list_icon
-    "icons/list/png/#{category&.kind || 'blank'}-#{icon_color}-22px.png"
+  def self.supporters_sql(comp = '<')
+    Arel.sql(<<~SQL.squish)
+      (SELECT COUNT(s.id) FROM supporter s WHERE s.issue_id = issue.id) #{comp} #{Settings::Vote.min_requirement}
```

Comment:
Tabellen-Namen bitte nicht fest einschreiben. Bitte in Supporter.table_name ändern.

Reply:
Insgesamt bitte in selbstgeschriebenem SQL analog zu `app/jobs/calculate_average_turnaround_time_job.rb` und der Rails-Vorgehensweise die Attribute und Tabellen im SQL quoten (`"`).